backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/MultiplicityRangeImpl.java

```diff
@@ -85,13 +93,25 @@ public Expression getLowerBound() {
     /**
      * <!-- begin-user-doc -->
      * <!-- end-user-doc -->
-     * @generated
+     * @generated NOT
      */
     public Expression basicGetLowerBound() {
-        // TODO: implement this method to return the 'Lower Bound' reference
-        // -> do not perform proxy resolution
-        // Ensure that you remove @generated or mark it @generated NOT
-        return null;
+        List<Expression> expressions = getOwnedElement().stream()
+                .filter(Expression.class::isInstance)
+                .map(Expression.class::cast)
+                .collect(Collectors.toList());
+
+        if (expressions.isEmpty()) {
+            return null;
+        }
+
+        if (expressions.get(0) instanceof LiteralInfinity) {
+            LiteralInteger literal = SysmlFactoryImpl.eINSTANCE.createLiteralInteger();
+            literal.setValue(0);
+            return literal;
+        }
+
+        return expressions.get(0);
```

Comment:
I think this code is not compliant whit checkstyle. Only 2 returns are allowed

Reply:
done